client/src/modules/prices/prices.html

```diff
@@ -44,6 +44,13 @@
                     <i class="fa fa-pencil"></i> <span translate>FORM.BUTTONS.EDIT</span>
                   </a>
                 </td>
+                <td width='50'>
+                  <a class="btn btn-xs btn-default" id="price_list_print_{{ $index+1 }}"
```

Comment:
Do you need the `id` here?  If you do, it would be much better to use the price list's `uuid` instead of the `$index`.  It will make testing easier in the future, since if we add more price lists, this ID might change.  If it's the price list's `uuid`, it will not.